kubectl-rook-ceph.sh

```diff
@@ -27,24 +27,26 @@ function print_usage() {
+  echo "  -n, --namespace='rook-ceph'                                 : the namespace of the CephCluster"
+  echo "  -o, --operator-namespace='rook-ceph'                        : the namespace of the rook operator"
+  echo " --context=<context_name>                                     : the name of the Kubernetes context to be used"
 
   echo "COMMANDS"
-  echo "  ceph <args>                               : call a 'ceph' CLI command with arbitrary args"
-  echo "  rbd <args>                                : call a 'rbd' CLI command with arbitrary args"
+  echo "  ceph <args>                                                 : call a 'ceph' CLI command with arbitrary args"
+  echo "  rbd <args>                                                  : call a 'rbd' CLI command with arbitrary args"
   echo "  operator <subcommand>..."
-  echo "    restart                                 : restart the Rook-Ceph operator"
-  echo "    set <property> <value>                  : Set the property in the rook-ceph-operator-config configmap."
-  echo "  mons                                      : output mon endpoints"
+  echo "    restart                                                   : restart the Rook-Ceph operator"
+  echo "    set <property> <value>                                    : Set the property in the rook-ceph-operator-config configmap."
+  echo "  mons                                                        : output mon endpoints"
   echo "  rook <subcommand>..."
-  echo "    version                                 : print the version of Rook"
-  echo "    status                                  : print the phase and conditions of the CephCluster CR"
-  echo "    status all                              : print the phase and conditions of all CRs"
-  echo "    status <CR>                             : print the phase and conditions of CRs of a specific type, such as 'cephobjectstore', 'cephfilesystem', etc"
-  echo "    purge-osd <osd-id> [--force]            : Permanently remove an OSD from the cluster. Multiple OSDs can be removed with a comma-separated list of IDs."
+  echo "    version                                                   : print the version of Rook"
+  echo "    status                                                    : print the phase and conditions of the CephCluster CR"
+  echo "    status all                                                : print the phase and conditions of all CRs"
+  echo "    status <CR>                                               : print the phase and conditions of CRs of a specific type, such as 'cephobjectstore', 'cephfilesystem', etc"
+  echo "    purge-osd <osd-id> [--force]                              : Permanently remove an OSD from the cluster. Multiple OSDs can be removed with a comma-separated list of IDs."
+  echo " create-debug [--name deployment] [--image <alternate-image>] : Make the OSD or mon in debug mode"
```

Comment:
I feel like `create-debug` is similar in scope to `purge-osd` which is under the `rook` subcommand. Should this move under that? @travisn ?

Reply:
> I feel like `create-debug` is similar in scope to `purge-osd` which is under the `rook` subcommand. Should this move under that? @travisn ?

What about `start-debug` and `stop-debug`?

Reply:
Instead of increasing the indentation of all the commands, how about moving the `--image` param to the next line? We will surely have commands with multiple args in the future, so this seems like a good pattern to start.